x-pack/plugins/rollup/public/crud_app/store/actions/change_job_status.js

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { toastNotifications } from 'ui/notify';
+import {
+  startJobs as sendStartJobsRequest,
+  stopJobs as sendStopJobsRequest,
+} from '../../services';
+import { refreshJobs } from './refresh_jobs';
+
+export const startJobs = (jobIds) => async (dispatch) => {
+  try {
+    await sendStartJobsRequest(jobIds);
+  } catch (error) {
+    return toastNotifications.addDanger(error.data.message);
```

Comment:
Thanks for handling errors! I'm not confident that errors will always have a `data.message` property though, we should probably use a helper of some sort that uses that when available, and defaults to something like `error.stack` when it isn't...

Reply:
https://github.com/elastic/kibana/issues/24777